src/ui/components/SecondaryToolbox/index.tsx

```diff
@@ -157,12 +161,22 @@ function SecondaryToolbox({
         </header>
       )}
       <Redacted className="secondary-toolbox-content text-xs">
-        {selectedPanel === "network" && <ConnectedRequestTable />}
-        {selectedPanel === "console" ? <ConsolePanel /> : null}
-        {selectedPanel === "inspector" ? <InspectorPanel /> : null}
-        {showReact && hasReactComponents && selectedPanel === "react-components" ? (
+        {/* {selectedPanel === "network" && <NetworkMonitor />} */}
+        <div className={classNames("h-full", { hidden: selectedPanel !== "network" })}>
+          <NetworkMonitor />
+        </div>
+        {/* {selectedPanel === "console" ? <ConsolePanel /> : null} */}
+        <div className={classNames("h-full", { hidden: selectedPanel !== "console" })}>
+          <ConsolePanel />
+        </div>
+        {/* {selectedPanel === "inspector" ? <InspectorPanel /> : null} */}
+        <div className={classNames("h-full", { hidden: selectedPanel !== "inspector" })}>
+          <InspectorPanel />
+        </div>
+        {/* {selectedPanel === "react-components" ? <ReactDevtoolsPanel /> : null} */}
+        <div className={classNames("h-full", { hidden: selectedPanel !== "react-components" })}>
           <ReactDevtoolsPanel />
-        ) : null}
+        </div>
```

Comment:
this code is less dark, but probably one of those things you just have to try and see how it feels